specification/monitor/resource-manager/Microsoft.Insights/stable/2021-05-01/metrics_API.json

```diff
@@ -0,0 +1,803 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "MonitorManagementClient",
+    "version": "2021-05-01"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Insights/metrics": {
+      "get": {
+        "tags": [
+          "Metrics"
+        ],
+        "operationId": "MultiResourceMetrics_List",
```

Comment:
A seemingly more common naming pattern would be `Metrics_ListAtSubscriptionScope`

Reply:
I've incorporated this feedback